.github/workflows/deploy.yml

```diff
@@ -31,10 +31,11 @@ on:
         required: true
         options:
           - dev
-      version:
-        description: Image version
-        type: string
-        required: true
+          - prod
+      # version:
+      #   description: Image version
+      #   type: string
+      #   required: true
```

Comment:
Why is this commented out?

Reply:
uncommented now. (lost them during my squash)